src/components/store/types/Store.ts

```diff
@@ -0,0 +1,28 @@
+import { Action } from './Action';
+import { EditorState } from './EditorState';
+
+/**
+ * Store type contains functions for use it
+ */
+export interface Store {
+  /**
+   * Function for subscribing on state changes
+   *
+   * @param listener - function, that will execute every state change
+   *
+   * @returns {() => void} unsubscribe function
+   */
+  subscribe: (listener: () => void) => (() => void);
```

Comment:
Why there is no ability to subscribe to the specific action?

Reply:
Cause I used redux as a reference. Subscribe function is used for subscribing to each state change.